modules/AOS_PDF_Templates/vardefs.php

```diff
@@ -41,6 +41,16 @@
 	'table'=>'aos_pdf_templates',
 	'audited'=>true,
 	'fields'=>array (
+        'description' =>
+            array (
+                'name' => 'description',
+                'vname' => 'LBL_DESCRIPTION',
+                'type' => 'text',
+                'comment' => 'Full text of the note',
+                'rows' => 6,
+                'cols' => 80,
+                'dbType' => 'mediumtext',
```

Comment:
I think this may cause issues with mssql as medium text is not supported see:

include/database/MssqlManager.php line 127 $type_map.

It may be a solution to change it to longtext and do some testing in mssql.